src/test/java/org/humancellatlas/ingest/project/ProjectFilterTest.java

```diff
@@ -254,11 +368,20 @@ void lookup_by_uuid() {
                 .containsExactly(project1);
     }
 
-
-
     @Test
     void all_args_constructor() {
```

Comment:
The "all args" constructor is used by Spring when it routes the web request to the controller.
I added this test to make sure we have one.
Not sure it is the most elegant way.